frontend/src/app/application-forms/tree-application-form/tree-application-form.component.html

```diff
@@ -1,19 +1,28 @@
 <section id="application" class="usa-section">
   <div class="usa-grid">
+    <nav class="breadcrumb">
```

Comment:
Our page titles are too long to be in a breadcrumb.  The standard way to deal with this is to create an alternate "short" title, which is what we are doing now though it isn't standardized other than in the breadcrumb.  If we made the page titles the "short" versions then they could match and the h1 on the page would be the only place with the "long" title.  Does that seem ok @lauraGgit ?

Reply:
also we can make a component, but there are some issues with how to pass the links in the various breadcrumbs.  we'd really have to make a db or something - I'm not sure how to proceed with that but I'll take a look.

Reply:
i'm ok with short titles that makes sense.
it did a quick scan and it seems like there are anuglar plugins for this. I think we would want to avoid this being db driven.